src/main/java/org/json/JSONObject.java

```diff
@@ -28,6 +28,9 @@
 import java.util.Set;
 import java.util.regex.Pattern;
 
+import static org.json.NumberConversionUtil.potentialNumber;
```

Comment:
Just import NumberConversionUtil, not the methods. Someone glancing at the code might not realize it is a static method in another class.